tests/test_ops/test_roi_align.py

```diff
@@ -40,6 +43,8 @@
 def _test_roialign_gradcheck(device, dtype):
     if not torch.cuda.is_available() and device == 'cuda':
         pytest.skip('test requires GPU')
+    if torch.is_mlu_available() and device == 'mlu':
```

Comment:
There is no `is_mlu_available` in torch without torch_mlu.

Reply:
It will be visible after another PR merged.

Reply:
Should we replace the `torch.is_mlu_available()` with `is_mlu()`

Reply:
Done